_layouts/post.html

```diff
@@ -34,11 +34,20 @@ <h1 class="post-title text-center">{{ page.title }}</h1>
       {% if page.meta %}
         &middot; {{ page.meta }}
       {% endif %}
-    </p>
+      {% if page.tags %}
```

Comment:
I did think of doing that inititally, but later thought that it would make more sense semantically to leave rendering logic outside of the partial.